staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -578,6 +586,19 @@ func cleanVerb(verb, suggestedVerb string, request *http.Request) string {
 	return OtherRequestMethod
 }
 
+//getVerbIfWatch additionally ensures that GET or List would be transformed to WATCH
```

Comment:
To avoid calling the same stuff multiple times let's change above:

cleanVerb:
```
var reportedVerb string
if suggestedVerb == "WATCH" || suggestedVerb == "WATCHLIST" {
	reportedVerb = "WATCH"
} else {
	reportedVerb = CleanVerb(verb, request)
}
```